cartridge/rpc.lua

```diff
@@ -42,7 +42,7 @@ local function member_is_healthy(uri, instance_uuid)
     local member = membership.get_member(uri)
     return (
         (member ~= nil)
-        and (member.status == 'alive')
+        and (member.status == 'alive' or member.status == 'suspect')
```

Comment:
I suggest to move such logic to some helper. To support consistency between:

- cartridge/cartridge/failover.lua
- cartridge/cartridge/roles/coordinator.lua
- cartridge/cartridge/rpc.lua

Reply:
While #1139 remains unresolved (the second part) I'd rather leave it as is. Moreover, all of them have slightly different API.